### PR TITLE
#trivial - Update f skeleton loader 2

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Changed
- *** Breaking change *** Update to `@use` and `@forward` SASS syntax
- Bump `sass` version.

### CODE REVIEW

**Review by FayeCarter — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "root",
-  "version": "7.13.0",
```
Comment (jamieomaguire) [inline package.json:3]: Should this be a major version bump?

**Review by jamieomaguire — DISMISSED** [review]

**Review by jamieomaguire — DISMISSED** [review]

**Review by jamieomaguire — DISMISSED** [review]

**Review by jamieomaguire — DISMISSED** [review]

**Review by oliversweb — DISMISSED** [review]

File: packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue [hunk 2]
```diff
@@ -56,13 +56,13 @@ $breakpoints: (medium: 520px, large: 960px);
     &:nth-child(1) {
         display: block;
     }
-    @include media('>medium') {
-        margin: 0 spacing(d) 0 0;
+    @include f.media('>520px') {
```
Comment (xander-marjoram) [inline packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue]: How come these have changed from breakpoints to pixels?

File: packages/components/molecules/f-skeleton-loader/src/components/skeletons/Heading.vue [hunk 3]
```diff
@@ -11,9 +11,11 @@ export default {
 
 
 <style lang="scss" module>
+@use '@justeat/fozzie/src/scss/fozzie' as f;
+
 .c-heading {
-    height: $skeleton-loader-heading-height;
+    height: common.$skeleton-loader-heading-height;
```
Comment (xander-marjoram) [inline packages/components/molecules/f-skeleton-loader/src/components/skeletons/Heading.vue:17]: How is the `common` module brought in?

Comment (jamieomaguire) [reply packages/components/molecules/f-skeleton-loader/src/components/skeletons/Heading.vue:17]: @xander-marjoram it's imported via the vue.config.js file for each component. We added some docs here to explain:
https://github.com/justeat/fozzie-components#reusable-styling-with-commonscss

File: packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue (hunk 2 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue]: Good spot. I actually have no idea how these have changed, i'm certain I didn't make this change. Will revert.

File: packages/components/molecules/f-skeleton-loader/src/components/skeletons/Heading.vue (hunk 3 above)
Comment (xander-marjoram) [reply packages/components/molecules/f-skeleton-loader/src/components/skeletons/Heading.vue:17]: Thanks, makes sense once you've read the webpack config!

File: packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue (hunk 2 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue]: Right I remember now. Further up in this file there's new values overriding the `$breakpoints` from within `fozzie` it was causing errors so I've just passed them in directly instead here.

Comment (ashleynolan) [reply packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue]: I'd probably assign the breakpoint value to a variable, as then you can name it so it's clear it's a specific breakpoint for this component (could call it `$bp-carouselMid` or something)

File: packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue [hunk 4]
```diff
@@ -56,13 +56,13 @@ $breakpoints: (medium: 520px, large: 960px);
...
+        margin: 0 f.spacing(d) 0 0;
         &:nth-child(2) {
             display: block;
         }
     }
-    @include media('>large') {
+    @include f.media('>960px') {
```
Comment (ashleynolan) [inline packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue]: Same here as above (`$bp-carouselWide`?)

**Review by jamieomaguire — APPROVED** [review]

File: packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue (hunk 4 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue]: Fixed.

**Review by BanksySan — APPROVED** [review]
Should we depreciate the previous versions?
https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions

**Comment by kevinrodrigues** [comment]
> Should we depreciate the previous versions? https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions

I'm not sure we usually follow a process of deprecating a single package or lower when bumping to breaking changes? I think we could if we wanted to but there's probably no real value in doing so. We're covering how to migrate over in-order to use these new versions in some documentation tickets so that should help.

**Comment by ashleynolan** [comment]
You only need to deprecate a package if it's been moved to another repo, or if it's been published by mistake. If it's just a new major version bump, it'll just take over the latest tag in the NPM registry.

**Pull request merged**